CommcareTranslationChecker/CommcareTranslationChecker.py

```diff
@@ -99,30 +159,32 @@ def getOutputCell(cell, wsOut):
 
     Output:
     Cell objects from wsOut corresponding to coordinates of cell 
-    '''
+    """
 
     return wsOut[cell.coordinate]
 
 
-def getNonLinguisticCharacterCount(val, additionalCharactersToCatch = None, characterList = None):
-    '''
-    Check a string for how many of each kind of non-linguistic character it contains and return a dictionary mapping character to count.
+def getNonLinguisticCharacterCount(val, additionalCharactersToCatch=None, characterList=None):
+    """
+    Check a string for how many of each kind of non-linguistic character it contains and
+    return a dictionary mapping character to count.
 
     Input:
     val(str): string to get counts from
     additionalCharactersToCatch(str [opt]): string of characters to append onto the characterList
-    characterList(str [opt]): string of characters considered non-linguistic. Defaults to CommcareTranslationChecker.NON_LINGUISTIC_CHARACTERS
+    characterList(str [opt]): string of characters considered non-linguistic.
```

Comment:
These are kind of weird names since they are passing in a string and not a `list`